dnf/base.py

```diff
@@ -1872,14 +1872,16 @@ def upgrade(self, pkg_spec, reponame=None):
     def upgrade_all(self, reponame=None):
         # :api
         if reponame is None and not self._update_security_filters and \
-                not self.repo_module_dict.list_module_version_installed():
+                not self.repo_module_dict.list_module_version_enabled():
```

Comment:
if you disable all modules, this will turn your system to non-modular instead of hiding modular contents